### ISSUE TITLE
Creating an account results in an error and doesn't complete. 

### ISSUE THREAD

**Issue by Bradymck** [district0x/ethlance#52]
See image for console error. 
![photo_2019-04-22_14-26-56](https://user-images.githubusercontent.com/4667076/56554067-b714ed80-654d-11e9-93f1-736492757484.jpg)

**Comment by Bradymck** [comment]
This appears to only happen in chrome. Forefox works properly.

**Comment by benzap** [comment]
So I was able to create a freelancer profile on the main website, however, there were a few other things that appear to be broken or limited: [here's a pic](https://i.imgur.com/MTg3Rxw.png)

- Can't link github profile, produces error "Pop up was closed" and closes the pop-up immediately after opening it
- Can't link linkedin profile, same deal.
- Can't seem to use metamask with a hardware ledger while in chrome. When attempting to create the profile, it just hangs, and a prompt on the ledger device never appears. I was only successful with a software wallet.

I was unable to test the hardware wallet in firefox, since it doesn't appear to be supported by metamask.

I'm not really sure how deep this issue goes, but a lot of the libraries used in the old ethlance are pretty old. I'm curious whether this would resolve some issues by version bumping, but i'm not 100% sure how to approach this problem.
